libfreeipmi/include/freeipmi/spec/ipmi-ipv6-address-status-spec.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
why a new file for this?  I dislike the idea of creating a new file for every tiny-subsection of the IPMI spec.